src/index.test.ts

```diff
@@ -308,3 +308,51 @@ describe('i18n', () => {
         expect(logger.log).toHaveBeenCalledTimes(callsLength + 1);
     });
 });
+
+const logger = {log: jest.fn()};
+describe('Incorrect instance initialization', () => {
+    beforeEach(() => {
+        i18n = new I18N({logger});
```

Comment:
For the sake of clearness let's create a new logger here as well, line below is not needed in that case

Reply:
Moved inside the case.